server/controllers/userControl.js

```diff
@@ -26,6 +26,14 @@ class UserControl {
             }
         )
     }
+    
+    static userLogin (req,resp) {
+        resp.status(200).send({
+            status: "success",
+            message: `Your are logged in as ${req.email}`,
```

Comment:
Please change this message to something that does not include the `req.email`.

Reply:
Okay